### PR TITLE
feat: sveltekit example

### PR DESCRIPTION
**Pull request by PuruVJ**
This adds sveltekit example using the new sveltekit vite plugin `@sveltejs/kit/vite`

### CODE REVIEW

File: examples/sveltekit/test/basic.test.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import Hello from '../src/lib/Hello.svelte'
+
+test('mount component', async() => {
+  const host = document.createElement('div')
+  host.setAttribute('id', 'host')
+  document.body.appendChild(host)
+  const instance = new Hello({ target: host, props: { count: 4 } })
+  expect(instance).toBeTruthy()
+  expect(host.innerHTML).toContain('4 x 2 = 8')
+  expect(host.innerHTML).toMatchSnapshot()
```
Comment (sheremet-va) [inline examples/sveltekit/test/basic.test.ts:12]: Maybe it's better to replace it with this:

```ts
expect(host).toMatchSnapshot()
```

This way generated snapshot will have nice formatting.

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: examples/sveltekit/test/basic.test.ts (hunk 1 above)
Comment (PuruVJ) [reply examples/sveltekit/test/basic.test.ts]: Done

**Comment by PuruVJ** [comment]
Updated for the latest sveltekit change

**Comment by antfu** [comment]
Tests are not passing, can you take a look? Thanks

**Comment by PuruVJ** [comment]
They are related to enzyme. I didn't even touch that, not sure whats going on there

**Comment by antfu** [comment]
You can try to rebase it to main

**Comment by antfu** [comment]
You can try to rebase it to main

**Comment by PuruVJ** [comment]
Weird, getting a lot of errors after running pnpm install after pulling latest changes.

Should I manually add these missing peer dependencies, or am I doing something wrong 🤔

```txt
docs
└─┬ vitepress 1.0.0-alpha.15
  └─┬ @docsearch/js 3.2.1
    └─┬ @docsearch/react 3.2.1
      └─┬ @algolia/autocomplete-preset-algolia 1.7.1
        └── ✕ missing peer @algolia/client-search@^4.9.1
Peer dependencies that should be installed:
  @algolia/client-search@^4.9.1  

examples/graphql
└─┬ @rollup/plugin-graphql 1.1.0
  └── ✕ missing peer rollup@"^1.20.0 || ^2.0.0"
Peer dependencies that should be installed:
  rollup@"^1.20.0 || ^2.0.0"  

examples/image-snapshot
└─┬ jest-image-snapshot 4.5.1
  └── ✕ missing peer jest@">=20 <=27"
Peer dependencies that should be installed:
  jest@">=20 <=27"  

examples/nextjs
└─┬ @vitejs/plugin-react 2.1.0
  └── ✕ missing peer vite@^3.0.0
Peer dependencies that should be installed:
  vite@^3.0.0  

examples/react-mui
├─┬ @testing-library/user-event 14.3.0
│ └── ✕ missing peer @testing-library/dom@>=7.21.4
├─┬ @emotion/react 11.9.3
│ ├── ✕ missing peer @babel/core@^7.0.0
│ └─┬ @emotion/babel-plugin 11.9.2
│   ├── ✕ missing peer @babel/core@^7.0.0
│   └─┬ @babel/plugin-syntax-jsx 7.18.6
│     └── ✕ missing peer @babel/core@^7.0.0-0
├─┬ @emotion/styled 11.9.3
│ └── ✕ missing peer @babel/core@^7.0.0
├─┬ @mui/x-date-pickers 5.0.0-beta.0
│ └── ✕ missing peer @mui/system@^5.4.1
└─┬ recharts 2.1.12
  └─┬ react-smooth 2.0.1
    └── ✕ missing peer prop-types@^15.6.0
Peer dependencies that should be installed:
  @babel/core@">=7.0.0 <8.0.0"   @mui/system@^5.4.1             @testing-library/dom@>=7.21.4  prop-types@^15.6.0             

examples/react-storybook
├─┬ @storybook/addon-essentials 6.5.9
│ ├── ✕ missing peer webpack@"*"
│ └─┬ @storybook/addon-docs 6.5.9
│   └─┬ babel-loader 8.2.5
│     └── ✕ missing peer webpack@>=2
├─┬ @storybook/builder-vite 0.1.37
│ ├── ✕ missing peer @storybook/core-common@">=6.4.3 || >=6.5.0-alpha.0"
│ └── ✕ missing peer @storybook/node-logger@">=6.4.3 || >=6.5.0-alpha.0"
├─┬ @storybook/react 6.5.9
│ └── ✕ missing peer require-from-string@^2.0.2
└─┬ @storybook/testing-react 1.3.0
  ├── ✕ missing peer @storybook/addons@>=6.4.0
  ├── ✕ missing peer @storybook/client-api@>=6.4.0
  └── ✕ missing peer @storybook/preview-web@>=6.4.0
Peer dependencies that should be installed:
  @storybook/addons@>=6.4.0                            @storybook/node-logger@">=6.4.3 || >=6.5.0-alpha.0"  webpack@>=2.0.0                                      
  @storybook/client-api@>=6.4.0                        @storybook/preview-web@>=6.4.0                       
  @storybook/core-common@">=6.4.3 || >=6.5.0-alpha.0"  require-from-string@^2.0.2                           

examples/react-testing-lib
└─┬ @testing-library/user-event 13.5.0
  └── ✕ missing peer @testing-library/dom@>=7.21.4
Peer dependencies that should be installed:
  @testing-library/dom@>=7.21.4  

examples/react-testing-lib-msw
├─┬ @testing-library/user-event 13.5.0
│ └── ✕ missing peer @testing-library/dom@>=7.21.4
└─┬ @apollo/client 3.6.9
  ├── ✕ missing peer graphql@"^14.0.0 || ^15.0.0 || ^16.0.0"
  ├─┬ @graphql-typed-document-node/core 3.1.1
  │ └── ✕ missing peer graphql@"^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0 || ^16.0.0"
  └─┬ graphql-tag 2.12.6
    └── ✕ missing peer graphql@"^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0 || ^16.0.0"
Peer dependencies that should be installed:
  @testing-library/dom@>=7.21.4                                       graphql@">=14.0.0 <15.0.0 || >=15.0.0 <16.0.0 || >=16.0.0 <17.0.0"
```

**Comment by sheremet-va** [comment]
> Should I manually add these missing peer dependencies, or am I doing something wrong 🤔

This is known errors, don't worry, if everything works fine.

**Comment by sheremet-va** [comment]
We have a very simplified example now. Feel free to push some changes there.

**Pull request closed without merging**